site/app/views/course/CourseMaterialsView.php

```diff
@@ -140,13 +142,28 @@ public function listCourseMaterials(array $course_materials_db) {
         ]);
     }
 
+    private function removeEmptyFolders(array &$course_materials): bool {
```

Comment:
Can we remove the `bool` here and just make this `void`? Unless we plan to utilize the returned value in some way, better to have less code.

Reply:
The bool/return false is needed for the return call on line 148 to determine if the folder was empty/only had empty folders in it